### PR TITLE
Add BlobGraph

### PR DESCRIPTION
**Pull request by 0xGh-st**
## Release Notes

- **New Features**
  - Introduced `BlobGraph` component for visualizing gas usage data in blockchain transactions.
  - Added `SecurityCouncil` component to display information about the Security Council for Layer 1 and Layer 2 networks.
  - New `BoxFrame`, `BoxFrameEx`, `ContentBox`, `CustomAccordion`, and `SubtitleBox` components for enhanced UI layout and content management.
  
- **Enhancements**
  - Integrated several new libraries for improved functionality, including axios for HTTP requests and Chart.js for data visualization.

- **Bug Fixes**
  - Adjusted import paths for better organization and clarity in the codebase.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,13 +18,19 @@
     "@mui/icons-material": "^6.1.1",
     "@mui/material": "^6.1.1",
     "@toolpad/core": "^0.7.0",
+    "axios": "^1.7.7",
+    "chart.js": "^4.4.4",
+    "cheerio": "^1.0.0",
+    "ethers": "^6.13.3",
```
[the remark this answers is not in the document]
Comment (gloomydumber) [reply package.json:24]: `cheerio` could be removed.
npm uninstall cheerio

**Review by gloomydumber — APPROVED** [review]
LGTM

**Comment by gdh8230** [comment]
ok

**Review by gdh8230 — APPROVED** [review]
ok

**Pull request merged**